Makefile

```diff
@@ -26,10 +26,13 @@ test-watch: compose_build
 lint: test_build
 	docker-compose -f docker-compose-test.yml run app flake8 .
 
-black: test_build
+black_check: test_build
+	docker-compose -f docker-compose-test.yml run app black -l 79 --check .
+
+black_fix: test_build
```

Comment:
I think it might be good to call this something besides `black_fix`. Maybe `code_format`? We may want to add other code formatters in the future (see #956). Alternatively I suppose we could have a command for each them.

Reply:
@mythmon Good idea, I'll make one called `code_format` that calls `black_fix` that way I can just refer to that in the docs and as we change or add more we don't need to update the docs.